submissions/HannaSyn/oop-exercise/index.js

```diff
@@ -0,0 +1,87 @@
+/* Refer to https://github.com/OleksiyRudenko/a-tiny-JS-world for the task details
+   Complete the below for code reviewers' convenience:
+
+   Code repository: _put repo URL here_
+   Web app: _put project's github pages URL here_
+   */
+
+// ======== OBJECTS DEFINITIONS ========
+// Define your objects here
+
+class Inhabitans {
+   constructor ( name, gender, friends, say, biologicalSpecies ){
+      this.name = name;
+      this.biologicalSpecies = biologicalSpecies;
+      this.gender = gender;
+      this.friends = friends;
+      this.say = say
+   }
+
+}
+
+class Human extends Inhabitans {
+   constructor( name, gender, friends, say, legs, hands ) {
+      super( name, gender, friends, say, 'human');
+      this.legs = legs;
+      this.hands = hands;
+   } 
+
+   speak() {
+      return ` ${this.say} I'm ${this.biologicalSpecies}! My name is ${this.name}, my gender is ${this.gender}, I have ${this.legs} legs and ${this.hands} hands and I have friends: ${this.friends}.`;
+   }
```

Comment:
This child class method handles properties of its super class. Let the super class do it itself.
You may re-use methods from superclass, the pattern is `super.methodName()`